src/__tests__/get-by-errors.js

```diff
@@ -39,6 +45,38 @@ cases(
   },
 )
 
+const GET_BY_QUERIES = [
+  'getByLabelText',
+  'getByPlaceholderText',
+  'getByText',
+  'getByAltText',
+  'getByTitle',
+  'getByRole',
+  'getByTestId',
+]
+const GET_ALL_BY_QUERIES = [
+  'getAllByLabelText',
+  'getAllByPlaceholderText',
+  'getAllByText',
+  'getAllByAltText',
+  'getAllByTitle',
+  'getAllByRole',
+  'getAllByTestId',
+]
```

Comment:
I think it might be better to use something like this:
```js
import { queries } from '@testing-library/dom'

const getQueries  = queries.filter(q => q.startsWith('get')
```

Reply:
Yes, that is much better. Will update.